test/build/resolve-imports/src/components/style.css

```diff
@@ -1 +1 @@
-body { color: red }
\ No newline at end of file
+@import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
We should also test for `@import url('…')` too, which is also valid syntax

Reply:
From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@import):

**Basic usage**

```css
@import 'custom.css';
@import url("chrome://communicator/skin/");
```

**Media queries**
```css
@import url("fineprint.css") print;
@import url("bluish.css") speech;
@import "common.css" screen;
@import url('landscape.css') screen and (orientation:landscape);
```

Reply:
I would say that we don't touch URL, and only touch the string-style import. Since we're actually changing spec behavior (`import 'foo/bar.css';` should import `bar.css` from package `foo`, instead of `./foo/bar.css`) I do like the idea of limiting this as much as possible. 

Regardless, we should document whatever behavior we land on.

Reply:
docs added